src/System.Private.CoreLib/shared/System/AggregateException.cs

```diff
@@ -440,26 +440,14 @@ public override string Message
             }
         }
 
-        /// <summary>
-        /// Creates and returns a string representation of the current <see cref="AggregateException"/>.
-        /// </summary>
-        /// <returns>A string representation of the current exception.</returns>
-        public override string ToString()
+        protected internal override string InnerExceptionToString(Exception? inner = null)
         {
-            StringBuilder text = new StringBuilder();
-            text.Append(base.ToString());
-
-            for (int i = 0; i < m_innerExceptions.Count; i++)
+            StringBuilder sb = StringBuilderCache.Acquire();
+            foreach(Exception ex in m_innerExceptions)
             {
-                text.AppendLine();
-                text.Append("---> ");
-                text.AppendFormat(CultureInfo.InvariantCulture, SR.AggregateException_InnerException, i);
-                text.Append(m_innerExceptions[i].ToString());
-                text.Append("<---");
-                text.AppendLine();
+                sb.Append(base.InnerExceptionToString(ex));
```

Comment:
Having removed all of the separators, what does an AggregateException now look like?

Reply:
See example linked in top post. I can also add one with an AggE wrapping an AggE.